File: springcloud/ohcprovider/src/main/java/com/gzst/shopping/service/BrandService.java
package com.gzst.shopping.service;


import com.gzst.shopping.model.Brand;
import com.gzst.shopping.utils.PageResult;

import java.util.List;

public interface BrandService {
    public List<Brand> findAllBrands();
    public PageResult<Brand> findPageBrands(Integer pageNo, Integer pageSize);
    public void add(Brand brand);
    public void update(Brand brand);
    public Brand findOne(Long id);
    public void delete(Long []ids);
}

File: springcloud/ohcprovider/src/main/resources/static/js/service/brandService.js
app.service('brandService',function ($http) {
    this.findAll=function () {
        return $http.get('../brand/findAll');
    }
    this.findPage=function (pageNo, pageSize) {
        return $http.get('../brand/findPage?pageNo='+pageNo+'&pageSize='+pageSize);
    }
    this.save=function (entity) {
        return $http.post('../brand/save',entity);
    }
    this.findOne=function (id) {
        return $http.get('../brand/findOne?id='+id);
    }
    this.deleteOne=function (id) {
        return $http.get('../brand/deleteOne?id='+id);
    }
    this.deleteMany=function (ids) {
        return $http.get('../brand/deleteMany?ids='+ids);
    }
})